eng/common/scripts/Verify-Links.ps1

```diff
@@ -239,9 +243,15 @@ function CheckLink ([System.Uri]$linkUri, $allowRetry=$true)
       }
     }
     catch {
-      $statusCode = $_.Exception.Response.StatusCode.value__
+      
+      $responsePresent = $_.Exception.psobject.Properties.name -contains "Response"
```

Comment:
FYI this fixes:

```
Checking 5 links found on page file:///C:/Users/kojamroz/repos/azure-sdk-docs-eng.ms/docs/onboard/requirements.md
VERBOSE: Checking link http://aka.ms/plrwiki...
VERBOSE: HTTP/1.1 HEAD with 0-byte payload
VERBOSE: received 42549-byte response of content type text/html
VERBOSE: Checking link file:///C:/Users/kojamroz/repos/azure-sdk-docs-eng.ms/docs/onboard/requirements.md...
VERBOSE: Checking link http://localhost:8080/docs/onboard/access.html...
VERBOSE: HTTP/1.1 HEAD with 0-byte payload
VERBOSE: HTTP/1.1 GET with 0-byte payload
Verify-Links.ps1: The property 'Response' cannot be found on this object. Verify that the property exists.
```

Reply:
With these changes can we now set `Set-StrictMode -Version 4` in this script?

Reply:
Let me do it in a separate PR.

Reply:
@weshaggard I set it to 3.0 as 4.0 has no meaning, per these links:

- https://github.com/MicrosoftDocs/PowerShell-Docs/pull/7334
- https://github.com/MicrosoftDocs/PowerShell-Docs/pull/7334/files#diff-b737be4f59ebf2c3ffd45afc9eeea15de29843e63270adccf9ba4171f9a675f8

I have also updated some parameter docs.